Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -92,24 +100,156 @@ protected virtual void OnShellPropertyChanged(object sender, PropertyChangedEven
         {
             if (e.PropertyName == Shell.FlyoutHeaderBehaviorProperty.PropertyName)
                 UpdateFlyoutHeaderBehavior();
-            else if (e.PropertyName == Shell.FlyoutBackgroundColorProperty.PropertyName)
-                UpdateFlyoutBackgroundColor();
+            else if (e.IsOneOf(
+				Shell.FlyoutBackgroundColorProperty, 
+				Shell.FlyoutBackgroundImageProperty, 
+				Shell.FlyoutBackgroundImageAspectProperty))
+                UpdateFlyoutBackground();
         }
 
-        protected virtual void UpdateFlyoutBackgroundColor()
-        {
-            var color = _shellContext.Shell.FlyoutBackgroundColor;
-            if (_defaultBackground == null && color.IsDefault)
-                return;
-
-            if (_defaultBackground == null)
-                _defaultBackground = _rootView.Background;
-
-            if (color.IsDefault)
-                _rootView.Background = _defaultBackground;
-            else
-                _rootView.Background = new ColorDrawable(color.ToAndroid());
-        }
+		//Size AspectFit(Size aspectSize, Size boundingSize)
```

Comment:
Why all this commented code?

Reply:
This code was the first implementation of the bgimage using drawable, but there were difficulties with the placement of the image.